docs/release/release_notes.md

```diff
@@ -3,7 +3,7 @@
 We are pleased to announce the release of DAOS version 2.0.
```

Comment:
Just noticed this, but don't we want this to say "2.0.1" versus "2.0"?

Reply:
i'd keep this generic at 2.0, the patchlevels are on the headings following this toplevel section